CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to Singleton Java client
+
+
+
+## Welcome
+
+Welcome to Singleton Java client! This guide provides information on filing issues and guidelines for open source contributors. **Please leave comments / suggestions if you find something is missing or incorrect.**
+
+## Get Started
+
+* Fork Repository
+Go to [Singleton repository](https://github.com/vmware/singleton) and click 'Fork' on top of page.
+
+* Download the forked repository to local
+
+    ``` 
+    git clone --branch g11n-java-client https://github.com/{github_user_id}/singleton.git g11n-java-client
+    ```
+
+* Set Signature, setup CI Pipeline 
+  Please refer to the [general guideline](https://github.com/vmware/singleton/blob/master/CONTRIBUTING.md).
+  Signature is mandatory for the commits, please make sure to get it worked. If not, your PR won't be able to merge.
+
+## Develop, Build and Test
+### Create your branch
+Changes should be made on your own fork in a new branch. The branch should be named g11n-java-client-XXX-description where XXX is the number of the issue. Follow below steps to create a new branch.
+* Add vmware/singleton as a remote repository
+
+  ```
+  git remote add vmware https://github.com/vmware/singleton.git

```

Comment:
'vmware' branch looks like not appropriate though you can add a random name, please use other more readable name.